src/base/emu-i386/simx86/interp.c

```diff
@@ -933,10 +933,12 @@ intop3b:		{ int op = ArOpsFR[D_MO(opc)];
 
 /*9c*/	case PUSHF: {
 			if (V86MODE() && (IOPL<3)) {
-			    CODE_FLUSH();
 			    /* virtual-8086 monitor */
-			    if (!(TheCPU.cr[4] & CR4_VME))
+			    if (!(TheCPU.cr[4] & CR4_VME)) {
+				CODE_FLUSH();
 				goto not_permitted;	/* GPF */
+			    }
+			    CODE_FLUSH();
```

Comment:
So just move it before `if`
to avoid duplication?

Reply:
It's just GitHub being a bit confusing showing the final result here as well for me

Reply:
Next patch is a bit larger,
could you please comment
there on the exact line, so
that I see it?